owtf/install/install.py

```diff
@@ -70,158 +62,54 @@ def install_in_directory(directory, command):
-
-
-def finish():
-    Colorizer.success("[*] Finished!")
-    Colorizer.info("[*] Start OWTF by running 'cd path/to/pentest/directory; ./path/to/owtf.py'")
-
-
-def setup_pip():
-    # Installing pip
-    Colorizer.info("[*] Installing pip")
-    directory = "/tmp/owtf-install/pip/%s" % os.getpid()
-    command = 'command -v pip2 >/dev/null || { wget --user-agent="Mozilla/5.0 (X11; Linux i686; rv:6.0) Gecko/20100101'\
-        ' Firefox/15.0" --tries=3 https://bootstrap.pypa.io/get-pip.py; sudo python get-pip.py;}'
-    install_in_directory(os.path.expanduser(directory), command)
-    Colorizer.info("[*] Installing required packages for pipsecure")
-    run_command("sudo pip2 install --upgrade pyopenssl ndg-httpsclient pyasn1")
-
-
-def install(cmd_arguments):
-    """Perform installation of OWTF Framework. Wraps around all helper methods made in this module.
+    """Checks if this is a Debian like linux distribution
 
-    :param cmd_arguments:
+    :return: `
+    :rtype: `bool`
     """
-    args = parser.parse_args(cmd_arguments)
-    # User asked to select distro (in case it can't be automatically detected) and distro related stuff is installed
-    cp = ConfigParser.ConfigParser({"RootDir": root_dir, "Pid": pid})
-    cp.read(distros_cfg)
-
-    # Try get the distro automatically
-    distro, version, arch = platform.linux_distribution()
-    distro_num = 0
-    if "kali" in distro.lower():
-        distro_num = 1
-    elif is_debian_derivative():
-        distro_num = 2
-
-    if distro_num != 0:
-        Colorizer.info("[*] %s has been automatically detected... " % distro)
-        Colorizer.normal("[*] Continuing in auto-mode")
-    elif (distro_num == 0) and args.no_user_input:
-        Colorizer.info("[*] Cannot auto-detect a supported distro...")
-        Colorizer.normal("[*] Continuing in auto-mode with the core installation...")
-    else:
-        # Loop until proper input is received
-        while True:
-            print("")
-            for i, item in enumerate(cp.sections()):
-                Colorizer.warning("(%d) %s" % (i + 1, item))
-            Colorizer.warning("(0) My distro is not listed :( %s" % distro)
-
-            num_input = raw_input("Select a number based on your distribution : ")
-            try:
-                if int(num_input) <= len(cp.sections()):
-                    distro_num = int(num_input)
-                    break
-                else:
-                    print("")
-                    Colorizer.warning("[!] Invalid number - not a supported distro")
-                    continue
-            except ValueError:
-                print("")
-                Colorizer.warning("[!] Invalid Number specified")
-                continue
-
-    # Install distro specific dependencies and packages needed for OWTF to work
-    if distro_num != 0:
-        run_command(cp.get(cp.sections()[int(distro_num) - 1], "install"))
+    compatible_value = os.system("which apt-get >> /dev/null 2>&1")
```

Comment:
Likely a stupid remark but this could be abused by a malicious user with `PATH` pointing to a custom directory, in the case OWTF is run with setuid.

Should we use absolute path instead?